src/content.js

```diff
@@ -14,7 +14,7 @@ function groupByRepo(events) {
 	return Array.from(events).reduce((repos, originalEvent) => {
 		const icon = originalEvent.querySelector('svg');
 		const actorEl = originalEvent.querySelector('.title a:first-child');
-		const eventEl = originalEvent.querySelector('.title a:nth-child(2)');
+		const eventEl = originalEvent.querySelector('.title a:last-child');
```

Comment:
This caused a bug when occurs element with one child. I tried to understand how this case could have been born, but could not. =(